From f9035d3fa5483f3b7a313c7cfd43c034c6660e1f Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 19 Jul 2018 17:35:37 +0800
Subject: [PATCH 274/706] drm: cdn_hdp: Get pixelClockFreq from vic_table
 directly

This patch makes pixelClockFreq get from vic_table directly.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 23a9f7c1d3b8ecc66a55df37a1942a9300a3621b)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/cdn_hdp/API_DPTX.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/hdp/cdn_hdp/API_DPTX.c b/drivers/gpu/drm/imx/hdp/cdn_hdp/API_DPTX.c
index e071e736f661..716c265a9495 100644
--- a/drivers/gpu/drm/imx/hdp/cdn_hdp/API_DPTX.c
+++ b/drivers/gpu/drm/imx/hdp/cdn_hdp/API_DPTX.c
@@ -479,9 +479,7 @@ CDN_API_STATUS CDN_API_DPTX_Set_VIC(state_struct *state, VIC_MODES vicMode,
 		bitsPerPixelCalc = bitsPerPixel * 3;
 
 	/* KHz */
-	pixelClockFreq =
-	    CDN_API_Get_PIXEL_FREQ_KHZ_ClosetVal(vic_table[vicMode]
-						 [PIXEL_FREQ_KHZ], CDN_DPTX);
+	pixelClockFreq = vic_table[vicMode][PIXEL_FREQ_KHZ];
 
 	/* KHz */
 	min_link_rate = rate * 995;
-- 
2.17.1

